trustpoint/cmp/urls.py

```diff
@@ -7,6 +7,17 @@
 app_name = 'cmp'
 
 urlpatterns = [
+    path(
```

Comment:
Is this a leftover from the branch for SPS demo? I would prefer not adding back this format as the operation (`initialization`) actually comes last in the URL in the RFCs. I would probably rework the views to make the operation completely optional, since it can be inferred from the CMP message. Then we can also do `/.well-known/cmp/p/<str:domain_name>`